### PR TITLE
Feat mutation - add participants

### PR DESCRIPTION
**Pull request by Damien1208**

### CODE REVIEW

File: src/graphql/schema/Trip/resolvers.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      if (!user) throw new AuthenticationError();
+      const {
+        destination = { isDictated: false },
+        budget = { isDictated: false },
+        timeFrame = { isDictated: false },
+        participants
+      } = trip;
+
+      trip.participants = await findUserOrCreate(participants, User);
+      trip.participants = [user._id, ...trip.participants];
```
Comment (Bueggi) [inline src/graphql/schema/Trip/resolvers.js:28]: Why do you assign trip.participants twice with different contents? Do we need line 27 if it is not used?

File: src/graphql/schema/resolver-helpers.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+export const voters = ({ voters }, _, { User }) => users(voters, User);
+
+export const creator = ({ creator }, _, { User }) => User.findOne(creator);
+
+export const buildSuggestionsObj = (dimension, authToken) => {
+  if (dimension.suggestions && dimension.suggestions.length) {
+    dimension.suggestions = dimension.suggestions.map(suggestion => ({
+      ...suggestion,
+      voters: [authToken],
```
Comment (Bueggi) [inline src/graphql/schema/resolver-helpers.js:16]: Why are we storing the authToken here? If someone - who has not created the trip - votes for that suggestion we have mixed data in the array. Would be consistent here and use the userID

**Review by Bueggi — CHANGES_REQUESTED** [review]
Looks good so far!
2 Things to discuss about:
1) Assigning a value twice to the same property
2) Storing the Access Token in the participants-array

Also changed the merge branch from master to develop - Well spotted, @mrcknz

File: src/graphql/schema/Trip/resolvers.js (hunk 1 above)
Comment (wrthwhl) [reply src/graphql/schema/Trip/resolvers.js:28]: One assignment builds on the other. the second assignment basically adds the user that creates the trip into the participants array. could be put into one line but this way it is (admittedly only a little bit) more explicit.

File: src/graphql/schema/resolver-helpers.js (hunk 2 above)
Comment (wrthwhl) [reply src/graphql/schema/resolver-helpers.js:16]: `buildSuggestionsObj()` is only used in the creation of a new trip where I think the logic to always make the logged-in user always is the creator of the suggestions is sound, right?

File: src/data.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+const data = {
```
Comment (wrthwhl) [inline src/data.js]: this file is no longer used and should be removed, i think.

**Review by wrthwhl — CHANGES_REQUESTED** [review]
remove `src/data.js`

**Review by wrthwhl — APPROVED** [review]
lgtm

**Review by Bueggi — APPROVED** [review]

**Pull request merged**